### DISCUSSION TITLE
What is the benefit in performance and gain?

### DISCUSSION THREAD

**Discussion by alekslyse** [TheTimeWalker/wallpanel-android#133] [General]
Firstly, this is not to step on this project. I like it, but for Home Assistant the three common methods for a dashboard is HA Native App, HA via web and HA via fullykiosk.

Whare does wallpanel put itself in this stack? Is it more effective? Does it do anything that the other ones dont?

**Comment by TheQue42** [comment]
fullykiosk costs money, isnt that enough?

**Comment by piotrtobolski** [comment]
On top of that it uses MQTT for integration with HA and thanks to this is using local push instead of local polling and gets e.g. instant notifications for URL changes.
